packages/cache/src/LocalCacheStorage.ts

```diff
@@ -14,45 +13,28 @@ export class LocalCacheStorage extends CacheStorage {
     return path.join(this.internalCacheFolder, hash);
   }
 
-  protected async _fetch(
-    hash: string,
-    outputFolder: string | string[]
-  ): Promise<boolean> {
+  protected async _fetch(hash: string): Promise<boolean> {
     const localCacheFolder = this.getLocalCacheFolder(hash);
 
     if (!fs.pathExistsSync(localCacheFolder)) {
       return false;
     }
 
-    await Promise.all(
-      outputFolderAsArray(outputFolder).map(async folder => {
-        const locationInCache = path.join(localCacheFolder, folder);
-
-        if (!fs.pathExistsSync(locationInCache)) {
-          return;
-        }
-
-        await fs.mkdirp(folder);
-        await fs.copy(locationInCache, folder);
-      })
-    );
+    fs.readdirSync(localCacheFolder).forEach(fileOrFolder => {
```

Comment:
I think the sync version is going to cause a perf regression compared to the async version. Now, node cannot copy multiple files/folders in parallell, if it wants to. Instead, it will wait for each item to be copied before picking up the next.